src/components/pages/AdminTools/ManageUsers.js

```diff
@@ -47,6 +47,7 @@ const ManageUsersPage = props => {
       .then(res => {
         alert(`${profile.first_name}'s profile was deleted`);
         console.log(res.data);
+        window.location.reload();

```

Comment:
Doesn't this reload the whole page? Is there a reason for using this? If you just wanted to re-render the page with updated user list I would imagine you just need to call the axios get profiles to set state and trigger a re-render

Reply:
I agree. I think this was in the existing code base so it was getting reused but we should switch this over so the page does not rerender.